comfyui新版子图功能bug

导入工作流时报错：

# ComfyUI Error Report
## Error Details
- **Node ID:** N/A
- **Node Type:** N/A
- **Exception Type:** 由于重新加载工作流数据出错，加载被中止
- **Exception Message:** Error: [SubgraphNode.configure] No subgraph input found for input image
## Stack Trace
```
Error: [SubgraphNode.configure] No subgraph input found for input image
    at ComfyNode._internalConfigureAfterSlots (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:120024:33)
    at ComfyNode.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:106824:40)
    at ComfyNode.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:120018:11)
    at ComfyNode.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:277190:15)
    at LGraph.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:118098:34)
    at LGraph.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:282773:26)
    at LGraph.configure (https://wywsl6s819eey6jw-80.container.x-gpu.com/extensions/ComfyUI-Custom-Scripts/js/reroutePrimitive.js:14:29)
    at ComfyApp.loadGraphData (https://wywsl6s819eey6jw-80.container.x-gpu.com/assets/index-DSs9bFiu.js:283070:18)
    at async app.loadGraphData (https://wywsl6s819eey6jw-80.container.x-gpu.com/extensions/ComfyUI-Manager/components-manager.js:774:9)
```
## System Information
- **ComfyUI Version:** 0.3.49
- **Arguments:** main.py --listen 0.0.0.0 --port 80
- **OS:** posix
- **Python Version:** 3.10.12 (main, Jul  5 2023, 18:54:27) [GCC 11.2.0]
- **Embedded Python:** false
- **PyTorch Version:** 2.5.1+cu124
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 4090 D : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 25278087168
  - **VRAM Free:** 15662463696
  - **Torch VRAM Total:** 9697230848
  - **Torch VRAM Free:** 581122768

## Logs
```
2025-08-11T03:27:37.950699 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:37.950908 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:37.967545 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:37.967727 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:38.411006 - 
 90%|██████████████████████████████████████▋    | 18/20 [00:07<00:00,  4.34it/s]2025-08-11T03:27:38.416165 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:38.416349 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:38.861107 - 
 95%|████████████████████████████████████████▊  | 19/20 [00:07<00:00,  3.74it/s]2025-08-11T03:27:38.866479 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:38.866665 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:27:39.314561 - 
100%|███████████████████████████████████████████| 20/20 [00:08<00:00,  3.29it/s]2025-08-11T03:27:39.314809 - 
100%|███████████████████████████████████████████| 20/20 [00:08<00:00,  2.50it/s]2025-08-11T03:27:39.314860 - 
2025-08-11T03:27:39.315721 - Requested to load AutoencodingEngine
2025-08-11T03:27:39.351708 - loaded completely 10368.413106918335 159.87335777282715 True
2025-08-11T03:27:40.010300 - comfyui lumi batcher overwrite task done2025-08-11T03:27:40.010488 - 
2025-08-11T03:27:40.016127 - Prompt executed in 17.66 seconds
2025-08-11T03:31:05.132825 - got prompt
2025-08-11T03:31:05.163510 - [Deep Translator] Service: "GoogleTranslator"2025-08-11T03:31:05.163712 - 
2025-08-11T03:31:05.163805 - [Deep Translator] Proxy disabled or input field is empty!2025-08-11T03:31:05.163861 - 
2025-08-11T03:31:05.163985 - [Deep Translator] Authorization input field is empty!2025-08-11T03:31:05.164052 - 
2025-08-11T03:31:05.164113 - Deep Translator] Service detect language disabled! Services support: DeeplTranslator, QcriTranslator, LingueeTranslator, PonsTranslator, PapagoTranslator, BaiduTranslator, MyMemoryTranslator.
The selected service has its own way of detecting the language.
Property "detect_lang_api_key" in Authorization data is empty or incorrect!2025-08-11T03:31:05.164224 - 
2025-08-11T03:31:10.088714 - # 😺dzNodes: LayerStyle -> [1;32mImageScaleByAspectRatio V2 Processed 1 image(s).[m2025-08-11T03:31:10.088800 - 
2025-08-11T03:31:10.354762 - Requested to load FluxClipModel_
2025-08-11T03:31:11.984527 - loaded completely 13717.689358520507 4777.53759765625 True
2025-08-11T03:31:12.186441 - Unload Model:2025-08-11T03:31:12.186720 - 
2025-08-11T03:31:14.509693 -  - Clearing Cache...2025-08-11T03:31:14.509759 - 
2025-08-11T03:31:15.144572 - Requested to load Flux
2025-08-11T03:31:15.145951 - loaded completely 12873.402636184692 122.3087158203125 True
2025-08-11T03:31:15.147637 - 
  0%|                                                    | 0/20 [00:00<?, ?it/s]2025-08-11T03:31:15.153765 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:15.153936 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:15.597640 - 
  5%|██▏                                         | 1/20 [00:00<00:08,  2.22it/s]2025-08-11T03:31:15.602779 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:15.602969 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.039798 - 
 10%|████▍                                       | 2/20 [00:00<00:08,  2.25it/s]2025-08-11T03:31:16.046085 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.046267 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.496207 - 
 15%|██████▌                                     | 3/20 [00:01<00:07,  2.22it/s]2025-08-11T03:31:16.501311 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.501494 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.940404 - 
 20%|████████▊                                   | 4/20 [00:01<00:07,  2.23it/s]2025-08-11T03:31:16.945481 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.945673 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.962235 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:16.962422 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.407966 - 
 30%|█████████████▏                              | 6/20 [00:02<00:04,  2.99it/s]2025-08-11T03:31:17.413194 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.413394 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.430124 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.430307 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.870769 - 
 40%|█████████████████▌                          | 8/20 [00:02<00:03,  3.45it/s]2025-08-11T03:31:17.875884 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.876079 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.892644 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:17.892838 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.337159 - 
 50%|█████████████████████▌                     | 10/20 [00:03<00:02,  3.73it/s]2025-08-11T03:31:18.342248 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.342431 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.359025 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.359208 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.376307 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.376497 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.814890 - 
 65%|███████████████████████████▉               | 13/20 [00:03<00:01,  4.56it/s]2025-08-11T03:31:18.820003 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.820191 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.836774 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:18.836956 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.278628 - 
 75%|████████████████████████████████▎          | 15/20 [00:04<00:01,  4.48it/s]2025-08-11T03:31:19.283846 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.284024 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.300505 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.300690 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.742916 - 
 85%|████████████████████████████████████▌      | 17/20 [00:04<00:00,  4.43it/s]2025-08-11T03:31:19.748079 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:19.748258 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:20.191346 - 
 90%|██████████████████████████████████████▋    | 18/20 [00:05<00:00,  3.77it/s]2025-08-11T03:31:20.196453 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:20.196632 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:20.642611 - 
 95%|████████████████████████████████████████▊  | 19/20 [00:05<00:00,  3.31it/s]2025-08-11T03:31:20.648267 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:20.648490 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:31:21.092684 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  2.98it/s]2025-08-11T03:31:21.092907 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  3.36it/s]2025-08-11T03:31:21.092944 - 
2025-08-11T03:31:21.093839 - Requested to load AutoencodingEngine
2025-08-11T03:31:21.123239 - loaded completely 10368.413106918335 159.87335777282715 True
2025-08-11T03:31:21.772087 - comfyui lumi batcher overwrite task done2025-08-11T03:31:21.772257 - 
2025-08-11T03:31:21.777355 - Prompt executed in 16.63 seconds
2025-08-11T03:32:12.270328 - got prompt
2025-08-11T03:32:14.497021 - # 😺dzNodes: LayerStyle -> [1;32mImageScaleByAspectRatio V2 Processed 1 image(s).[m2025-08-11T03:32:14.497092 - 
2025-08-11T03:32:14.862033 - 
  0%|                                                    | 0/20 [00:00<?, ?it/s]2025-08-11T03:32:14.867528 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:14.867722 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:15.307140 - 
  5%|██▏                                         | 1/20 [00:00<00:08,  2.25it/s]2025-08-11T03:32:15.313306 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:15.313493 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:15.750153 - 
 10%|████▍                                       | 2/20 [00:00<00:07,  2.25it/s]2025-08-11T03:32:15.756672 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:15.756915 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.208013 - 
 15%|██████▌                                     | 3/20 [00:01<00:07,  2.22it/s]2025-08-11T03:32:16.213079 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.213275 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.653876 - 
 20%|████████▊                                   | 4/20 [00:01<00:07,  2.23it/s]2025-08-11T03:32:16.659308 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.659526 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.680623 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:16.680854 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.126992 - 
 30%|█████████████▏                              | 6/20 [00:02<00:04,  2.97it/s]2025-08-11T03:32:17.132312 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.132523 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.151096 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.151302 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.591411 - 
 40%|█████████████████▌                          | 8/20 [00:02<00:03,  3.43it/s]2025-08-11T03:32:17.596848 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.597047 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.615046 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:17.615239 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.059567 - 
 50%|█████████████████████▌                     | 10/20 [00:03<00:02,  3.71it/s]2025-08-11T03:32:18.064725 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.064926 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.081936 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.082133 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.099844 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.100039 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.542670 - 
 65%|███████████████████████████▉               | 13/20 [00:03<00:01,  4.53it/s]2025-08-11T03:32:18.547719 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.547927 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.564963 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:18.565149 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.007686 - 
 75%|████████████████████████████████▎          | 15/20 [00:04<00:01,  4.46it/s]2025-08-11T03:32:19.012851 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.013037 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.029588 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.029795 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.477538 - 
 85%|████████████████████████████████████▌      | 17/20 [00:04<00:00,  4.40it/s]2025-08-11T03:32:19.483337 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.483557 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.928683 - 
 90%|██████████████████████████████████████▋    | 18/20 [00:05<00:00,  3.75it/s]2025-08-11T03:32:19.934026 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:19.934225 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:20.380407 - 
 95%|████████████████████████████████████████▊  | 19/20 [00:05<00:00,  3.29it/s]2025-08-11T03:32:20.386663 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:20.386882 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:32:20.833061 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  2.97it/s]2025-08-11T03:32:20.833300 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  3.35it/s]2025-08-11T03:32:20.833338 - 
2025-08-11T03:32:21.497523 - comfyui lumi batcher overwrite task done2025-08-11T03:32:21.497755 - 
2025-08-11T03:32:21.501050 - Prompt executed in 9.22 seconds
2025-08-11T03:33:02.383826 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/photoswipe-lightbox.esm.min.js2025-08-11T03:33:02.383887 - 
2025-08-11T03:33:02.465582 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/photoswipe.min.css2025-08-11T03:33:02.465642 - 
2025-08-11T03:33:02.469562 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/pickr.min.js2025-08-11T03:33:02.469798 - 
2025-08-11T03:33:02.600934 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/juxtapose.css2025-08-11T03:33:02.601091 - 
2025-08-11T03:33:02.810341 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/classic.min.css2025-08-11T03:33:02.810399 - 
2025-08-11T03:33:02.816914 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/juxtapose.min.js2025-08-11T03:33:02.816988 - 
2025-08-11T03:33:02.821034 - /root/ComfyUI/custom_nodes/comfyui-mixlab-nodes/webApp/lib/model-viewer.min.js2025-08-11T03:33:02.821073 - 
2025-08-11T03:33:02.943794 - Feature flags negotiated for client 18a997d398ec4fd1ae76f2839ee7ab2f: {'supports_preview_metadata': True}
2025-08-11T03:33:03.912002 - [ERROR] An error occurred while retrieving information for the 'PromptComposerCustomLists' node.
2025-08-11T03:33:03.912292 - Traceback (most recent call last):
  File "/root/ComfyUI/server.py", line 634, in get_object_info
    out[x] = node_info(x)
  File "/root/ComfyUI/server.py", line 595, in node_info
    if issubclass(obj_class, _ComfyNodeInternal):
TypeError: issubclass() arg 1 must be a class

2025-08-11T03:33:03.912399 - [ERROR] An error occurred while retrieving information for the 'PromptComposerStyler' node.
2025-08-11T03:33:03.912502 - Traceback (most recent call last):
  File "/root/ComfyUI/server.py", line 634, in get_object_info
    out[x] = node_info(x)
  File "/root/ComfyUI/server.py", line 595, in node_info
    if issubclass(obj_class, _ComfyNodeInternal):
TypeError: issubclass() arg 1 must be a class

2025-08-11T03:33:03.912571 - [ERROR] An error occurred while retrieving information for the 'PromptComposerEffect' node.
2025-08-11T03:33:03.912683 - Traceback (most recent call last):
  File "/root/ComfyUI/server.py", line 634, in get_object_info
    out[x] = node_info(x)
  File "/root/ComfyUI/server.py", line 595, in node_info
    if issubclass(obj_class, _ComfyNodeInternal):
TypeError: issubclass() arg 1 must be a class

2025-08-11T03:33:04.085083 - []2025-08-11T03:33:04.085138 - 
2025-08-11T03:33:04.085169 - []2025-08-11T03:33:04.085194 - 
2025-08-11T03:33:38.969331 - got prompt
2025-08-11T03:33:38.978990 - Failed to validate prompt for output 318:
2025-08-11T03:33:38.979159 - * (prompt):
2025-08-11T03:33:38.979279 -   - Required input is missing: images
2025-08-11T03:33:38.979397 - * PreviewImage 318:
2025-08-11T03:33:38.979501 -   - Required input is missing: images
2025-08-11T03:33:38.979610 - Output will be ignored
2025-08-11T03:33:38.987324 - Failed to validate prompt for output 321:
2025-08-11T03:33:38.987486 - * FluxKontextImageScale 42:
2025-08-11T03:33:38.987606 -   - Required input is missing: image
2025-08-11T03:33:38.987715 - Output will be ignored
2025-08-11T03:33:38.989136 - Failed to validate prompt for output 193:
2025-08-11T03:33:38.989288 - Output will be ignored
2025-08-11T03:33:38.989423 - Failed to validate prompt for output 248:
2025-08-11T03:33:38.989540 - * (prompt):
2025-08-11T03:33:38.989642 -   - Required input is missing: images
2025-08-11T03:33:38.989774 - * PreviewImage 248:
2025-08-11T03:33:38.989877 -   - Required input is missing: images
2025-08-11T03:33:38.989977 - Output will be ignored
2025-08-11T03:33:38.990094 - Failed to validate prompt for output 301:
2025-08-11T03:33:38.990201 - * (prompt):
2025-08-11T03:33:38.990303 -   - Required input is missing: text
2025-08-11T03:33:38.990417 - * ShowText|pysssss 301:
2025-08-11T03:33:38.990520 -   - Required input is missing: text
2025-08-11T03:33:38.990623 - Output will be ignored
2025-08-11T03:33:39.034724 - [Deep Translator] Service: "GoogleTranslator"2025-08-11T03:33:39.035010 - 
2025-08-11T03:33:39.035090 - [Deep Translator] Proxy disabled or input field is empty!2025-08-11T03:33:39.035236 - 
2025-08-11T03:33:39.035322 - [Deep Translator] Authorization input field is empty!2025-08-11T03:33:39.035432 - 
2025-08-11T03:33:39.035521 - Deep Translator] Service detect language disabled! Services support: DeeplTranslator, QcriTranslator, LingueeTranslator, PonsTranslator, PapagoTranslator, BaiduTranslator, MyMemoryTranslator.
The selected service has its own way of detecting the language.
Property "detect_lang_api_key" in Authorization data is empty or incorrect!2025-08-11T03:33:39.035641 - 
2025-08-11T03:33:39.678466 - comfyui lumi batcher overwrite task done2025-08-11T03:33:39.678679 - 
2025-08-11T03:33:39.684422 - Prompt executed in 0.69 seconds
2025-08-11T03:36:32.826428 - got prompt
2025-08-11T03:36:32.858524 - [Deep Translator] Service: "GoogleTranslator"2025-08-11T03:36:32.858695 - 
2025-08-11T03:36:32.858813 - [Deep Translator] Proxy disabled or input field is empty!2025-08-11T03:36:32.858874 - 
2025-08-11T03:36:32.858974 - [Deep Translator] Authorization input field is empty!2025-08-11T03:36:32.859209 - 
2025-08-11T03:36:32.859324 - Deep Translator] Service detect language disabled! Services support: DeeplTranslator, QcriTranslator, LingueeTranslator, PonsTranslator, PapagoTranslator, BaiduTranslator, MyMemoryTranslator.
The selected service has its own way of detecting the language.
Property "detect_lang_api_key" in Authorization data is empty or incorrect!2025-08-11T03:36:32.859398 - 
2025-08-11T03:36:35.541603 - # 😺dzNodes: LayerStyle -> [1;32mImageScaleByAspectRatio V2 Processed 1 image(s).[m2025-08-11T03:36:35.541664 - 
2025-08-11T03:36:35.784847 - Requested to load FluxClipModel_
2025-08-11T03:36:37.337879 - loaded completely 13814.618130493163 4777.53759765625 True
2025-08-11T03:36:37.539840 - Unload Model:2025-08-11T03:36:37.539933 - 
2025-08-11T03:36:40.101207 -  - Clearing Cache...2025-08-11T03:36:40.101312 - 
2025-08-11T03:36:40.947770 - Requested to load Flux
2025-08-11T03:36:40.949164 - loaded completely 12875.447447433471 122.3087158203125 True
2025-08-11T03:36:40.950809 - 
  0%|                                                    | 0/20 [00:00<?, ?it/s]2025-08-11T03:36:40.957449 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:40.957620 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:41.393373 - 
  5%|██▏                                         | 1/20 [00:00<00:08,  2.26it/s]2025-08-11T03:36:41.401431 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:41.401599 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:41.838215 - 
 10%|████▍                                       | 2/20 [00:00<00:07,  2.25it/s]2025-08-11T03:36:41.843543 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:41.843744 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.295256 - 
 15%|██████▌                                     | 3/20 [00:01<00:07,  2.22it/s]2025-08-11T03:36:42.300560 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.300750 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.739485 - 
 20%|████████▊                                   | 4/20 [00:01<00:07,  2.23it/s]2025-08-11T03:36:42.744779 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.744955 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.762044 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:42.762424 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.207246 - 
 30%|█████████████▏                              | 6/20 [00:02<00:04,  2.99it/s]2025-08-11T03:36:43.212666 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.212865 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.231279 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.231463 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.670817 - 
 40%|█████████████████▌                          | 8/20 [00:02<00:03,  3.45it/s]2025-08-11T03:36:43.676904 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.677101 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.695120 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.695316 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.715946 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:43.716155 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.160220 - 
 55%|███████████████████████▋                   | 11/20 [00:03<00:02,  4.36it/s]2025-08-11T03:36:44.166041 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.166230 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.184739 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.185067 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.201717 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.201912 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.641908 - 
 70%|██████████████████████████████             | 14/20 [00:03<00:01,  4.97it/s]2025-08-11T03:36:44.647566 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.647770 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.664726 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:44.664923 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.109664 - 
 80%|██████████████████████████████████▍        | 16/20 [00:04<00:00,  4.75it/s]2025-08-11T03:36:45.116263 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.116447 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.133130 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.134699 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.578372 - 
 90%|██████████████████████████████████████▋    | 18/20 [00:04<00:00,  4.60it/s]2025-08-11T03:36:45.583783 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:45.583988 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:46.026274 - 
 95%|████████████████████████████████████████▊  | 19/20 [00:05<00:00,  3.90it/s]2025-08-11T03:36:46.032149 - Passing `txt_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:46.032340 - Passing `img_ids` 3d torch.Tensor is deprecated.Please remove the batch dimension and pass it as a 2d torch Tensor
2025-08-11T03:36:46.480376 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  3.39it/s]2025-08-11T03:36:46.480791 - 
100%|███████████████████████████████████████████| 20/20 [00:05<00:00,  3.62it/s]2025-08-11T03:36:46.480851 - 
2025-08-11T03:36:46.482169 - Requested to load AutoencodingEngine
2025-08-11T03:36:46.534787 - loaded completely 10368.413106918335 159.87335777282715 True
2025-08-11T03:36:47.188348 - comfyui lumi batcher overwrite task done2025-08-11T03:36:47.188525 - 
2025-08-11T03:36:47.191988 - Prompt executed in 14.35 seconds

```
## Attached Workflow
Please make sure that workflow does not contain any sensitive information such as API keys or passwords.
```
Workflow too large. Please manually upload the workflow from local file system.
```

## Additional Context
(Please add any additional context or steps to reproduce the error here)

┆Issue is synchronized with this [Notion page](https://www.notion.so/Issue-4905-comfyui-bug-24c6d73d36508147aae1f93810609d58) by [Unito](https://www.unito.io)

--- a/src/lib/litegraph/src/subgraph/SubgraphNode.ts
+++ b/src/lib/litegraph/src/subgraph/SubgraphNode.ts
@@ -266,10 +266,14 @@ export class SubgraphNode extends LGraphNode implements BaseLGraph {
       const subgraphInput = this.subgraph.inputNode.slots.find(
         (slot) => slot.name === input.name
       )
-      if (!subgraphInput)
-        throw new Error(
-          `[SubgraphNode.configure] No subgraph input found for input ${input.name}`
+      if (!subgraphInput) {
+        // Skip inputs that don't exist in the subgraph definition
+        // This can happen when loading workflows with dynamically added inputs
+        console.warn(
+          `[SubgraphNode.configure] No subgraph input found for input ${input.name}, skipping`
         )
+        continue
+      }
 
       this.#addSubgraphInputListeners(subgraphInput, input)
 

